config/webpack/webpack.client.babel.ts

```diff
@@ -145,7 +145,7 @@ export default {
     compress: true,
     disableHostCheck: true,
     historyApiFallback: true,
-    host: '0.0.0.0',
+    host: '127.0.0.1',
```

Comment:
It will break for people who work on remote machines. Why are these necessary?
It works for me with `0.0.0.0` on Windows 7 VM.
If it is really necessary, please make it configurable through an environment variable and add to `.env.example` file

Reply:
On both my windows machines, 0.0.0.0 didn't work.  127.0.0.1 is the standard localhost IP address.

Reply:
@eloisetaylor5693 Could you please patch and verify it for us?
Add and `.env` variable and set it to `127.0.0.1` by default. Refer to this variable in webpack config.  Those who need remote will change the var to `0.0.0.0`,
Thanks for your investigation!